codecov.yml

```diff
@@ -7,47 +7,43 @@ coverage:
     # Learn more at https://docs.codecov.io/docs/commit-status
     project:
       default:
-        threshold: 1% # allow this much decrease on project
+        target: auto
+        threshold: 1% # # allow this much decrease on project
```

Comment:
```suggestion
        threshold: 35% # # allow this much decrease on project
```

Reply:
Why 35% 👀, i think this percentage is too much

Reply:
Why it's too much?

Reply:
Okay, I am making 2%